### PR TITLE
[12.0][MIG] purchase_comment_template

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration from v10.

### CODE REVIEW

File: purchase_comment_template/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    @api.onchange('comment_template2_id')
+    def _set_note2(self):
+        comment = self.comment_template2_id
+        if comment:
+            self.note2 = comment.get_value(self.partner_id.id)
+
+    @api.onchange('partner_id')
+    def onchange_partner_id_purchase_comment(self):
+        if self.partner_id:
+            comment_template = self.partner_id.comment_template_id
```
Comment (daramousk) [inline purchase_comment_template/models/purchase_order.py]: `comment_template_id` has been renamed to `property_comment_template_id`

File: purchase_comment_template/views/purchase_order_view.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<odoo>
+
+    <record model="ir.ui.view" id="purchase_order_form_add_comment">
```
Comment (daramousk) [inline purchase_comment_template/views/purchase_order_view.xml:3]: This needs to be formatted with 4 spaced tabs, there are some formatting issues on the xml files.

File: purchase_comment_template/views/report_purchaseorder.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      inherit_id="purchase.report_purchaseorder_document">
+      <xpath expr="//div[hasclass('oe_structure')][1]" position="before">
+        <template id="purchase_css" inherit_id="web.layout">
+            <xpath expr="//head" position="inside">
+                <link rel="stylesheet"
+                    href="/purchase_comment_template/static/src/css/purchase.css" />
+            </xpath>
+        </template>
+      </xpath>
+      <xpath expr="//table[hasclass('table-sm')]" position="before">
```
Comment (daramousk) [inline purchase_comment_template/views/report_purchaseorder.xml:14]: There might be room for improvement for how this appears on the report, right now it stands on top.

**Review by daramousk — CHANGES_REQUESTED** [review]
Only one issue plus some comments.

File: purchase_comment_template/views/report_purchaseorder.xml (hunk 3 above)
Comment (MiquelRForgeFlow) [reply purchase_comment_template/views/report_purchaseorder.xml:14]: what do you propose?

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Just functional. Works as expected.

**Review by JoanSForgeFlow — APPROVED** [review]
Functional review done, LGTM :+1:

**Review by HviorForgeFlow — APPROVED** [review]
Functional Review :+1:

**Comment by AaronHForgeFlow** [comment]
Ready to merge IMHO.

**Pull request merged**